mreg_cli/api/models.py

```diff
@@ -523,6 +529,16 @@ def get_roles_with_atom(cls, name: str) -> list[Role]:
         data = get_list(cls.endpoint(), params={"atoms__name__exact": name})
         return [Role(**item) for item in data]
 
+    def add_atom(self, atom: str) -> bool:
+        """Add an atom to the role."""
+        resp = post(Endpoint.HostPolicyRoleAddAtom.with_params(self.name), name=atom)
+        return resp.ok if resp else False
+
+    def remove_atom(self, atom: str) -> bool:
+        """Remove an atom from the role."""
+        resp = delete(Endpoint.HostPolicyRoleRemoveAtom.with_params(self.name, atom))
+        return resp.ok if resp else False
+
```

Comment:
If we're not gonna be able to re-use the `create()` and `delete()` instance methods from `APIMixin`, we could consider making these classmethods.

Reply:
I like `delete()` (or "remove") to be an object method tbh, but `create` obviously can't be. :)

Reply:
I was referring to the `{add,remove}_atom` methods, but I don't think it's important either way.

Reply:
Oh, right, blind as a bat. Ignore me.